Mathlib/Tactic/Positivity/Core.lean

```diff
@@ -252,28 +251,48 @@ where `a` is a numeral. -/
 def compareHyp (e : Q($α)) (ldecl : LocalDecl) : MetaM (Strictness zα pα e) := do
   have e' : Q(Prop) := ldecl.type
```

Comment:
```suggestion
  have e' : Q(Prop) := ldecl.type
  have p : Q($e') := .fvar ldecl.fvarId
```
then you can drop the `p`s below

Reply:
it'd be nice if there was an inbuilt `Qq` way to do this, it feels a little type-unsafe